icubam/www/handlers/show.py

```diff
@@ -11,6 +14,9 @@ def initialize(self, db):
 
   def get_icu_data(self):
     df = self.db.get_bedcount()
+    df['since_update'] = df.update_ts.apply(update.time_ago)
+    n_covid_tot = df['n_covid_free'] + df['n_covid_occ']
+    df.insert(1, 'n_covid_tot', n_covid_tot)
```

Comment:
And maybe one comment on what exactly this does, pandas can by a bit cryptic sometimes.